Gemfile.lock

```diff
@@ -319,9 +319,9 @@ GEM
       nokogiri (>= 1.5.11, < 2.0.0)
     formatador (0.2.5)
     front_matter_parser (0.2.1)
-    fugit (1.3.8)
+    fugit (1.4.1)
```

Comment:
dependency of `sidekiq-cron`, used to parse the cron syntax, nothing to report https://github.com/floraison/fugit/blob/master/CHANGELOG.md#fugit-141-released-2020-11-25